docs/ocaml/00-outline.md

```diff
@@ -0,0 +1,143 @@
+1. Intro
+  - Why Functional Programming?
+    - mutability
+    - encapsulation
+    - functional programming in Java is tedious (but possible)
+  - Goal: Introduce what we will build at the very end (after async)
+    - and show demo possible (or a snippet of code)
+
+2. Basics
+  - Utop (how to run)
```

Comment:
Maybe talk about how to install OCaml and Utop. It would be fine to give everybody this link:

http://dev.realworldocaml.org/install.html

Reply:
I think the goal would be to have everyone setup ocaml and utop before the demo. Most people will already have that on their machine anyway.